cosmo/monitors/data_models.py

```diff
@@ -247,3 +247,61 @@ def get_new_data(self):
                                            table_request=table_request)
 
         return data_results
+
+
+class CorrtagDataModel(BaseDataModel):
+    """Datamodel for all NUV Dark files."""
+    files_source = FILES_SOURCE
+    subdir_pattern = '?????'
+
+    def get_new_data(self):
+        header_request = {
+            0: ['ROOTNAME'],
+            1: [
+                'EXPSTART',
+                'EXPTIME',
+                'EXPTYPE',
+                'DETECTOR'
+                ]
+            }
+
+        table_request = {
+            1: [
+                'TIME',
+                'XCORR',
+                'YCORR'
+                ],
+            3: [
+                'LATITUDE',
+                'LONGITUDE'
+                ]
+
+            }
```

Comment:
```suggestion
        header_request = {
            0: ['ROOTNAME'],
            1: ['EXPSTART', 'EXPTIME', 'EXPTYPE', 'DETECTOR']
            }

        table_request = {
            1: ['TIME', 'XCORR', 'YCORR'],
            3: ['LATITUDE', 'LONGITUDE']
            }
```
No need to expand those lists since they're short enough.

Reply:
lol cool